CHANGELOG.md

```diff
@@ -136,6 +136,7 @@ Please see https://github.com/rapidsai/cuml/releases/tag/branch-0.18-latest for
 - PR #2916: Add SKLearn multi-class GBDT model support in FIL
 
 ## Improvements
+- PR #3014: NVTX Markers for RF and RF-backend
```

Comment:
please make sure to remove this entry since `0.16` has already been released. Manual changelog entries are no longer required, so you can just omit any changes to this file entirely.

Reply:
my bad @ajschmidt8 !, taken it out ✌🏾